sap-s4hana-cloud/2.2/antora.yml

```diff
@@ -0,0 +1,16 @@
+name: sap-s4hana-cloud-connector
```

Comment:
Should this be changed to remove cloud?

Reply:
This one isn't user facing and it's in a registry so it would involve infrastructure to change this. Same as the URLs will still have "cloud" but I don't want to mess with this right now.